cfpb/hmda-platform
# Generate and publish disclosure reports

Local testing instructions:
* Verify that AWS environment variables are set correctly
* Run the backend, and get a submission through to the "sign" step.  I recommend using very small files initially.  A file with 30 different MSAs will take roughly an hour to publish the 1000+ reports.
* After signing, the report generation will begin.  This can be verified in the logs at the INFO level.
* Check the S3 bucket to verify that reports are being published at `/reports/disclosure/{instId}/{msa}/{reportId}.txt`
* To test the publication again without uploading another file, POST to the admin API at `/disclosure/{instId}/{year}/{submissionId}`

Things left to do:
* Try to implement more parallelism to speed up the process without overloading Cassandra
* Documentation
* Tests

Closes #1395
